src/ViewInstance.js

```diff
@@ -900,6 +900,7 @@ class ViewInstance extends React.Component {
       },
     ];
     const isInstanceLoading = this.state.isLoading || !instance || isCentralTenantPermissionsLoading;
+    const keyInStorageToHoldingsAccsState = ['holdings'];
```

Comment:
It looks like it could be a constant which can be imported and used directly in `<HoldingAccordion>` without passing it through the component tree. Is there any reason to have it passed as a prop?